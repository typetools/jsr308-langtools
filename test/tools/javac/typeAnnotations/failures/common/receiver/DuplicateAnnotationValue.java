/*
 * @test /nodynamiccopyright/
 * @bug 6843077 6919944
 * @summary check for duplicate annotation values in receiver
 * @author Mahmood Ali
 * @compile/fail/ref=DuplicateAnnotationValue.out -XDrawDiagnostics -source 1.8 DuplicateAnnotationValue.java
 */
class DuplicateAnnotationValue {
  void test(@A(value = 2, value = 1) DuplicateAnnotationValue this) { }
}

@interface A { int value(); }
