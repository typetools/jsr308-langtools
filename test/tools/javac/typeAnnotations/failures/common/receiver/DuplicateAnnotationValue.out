DuplicateAnnotationValue.java:31:37: compiler.err.duplicate.annotation.member.value: value, A
1 error
